### PR TITLE
Packaging

### PR DESCRIPTION
**Pull request by MaxBlesch**
We want to polish the appearance of robupy and make it conda installable.

### CODE REVIEW

**Comment by MaxBlesch** [comment]
So there is everything in there except the Codecov badge. I can create this once codecov ran in master. I left the zenodo badge in there! Also I adjusted the citing of robupy according to respy. In our paper we cited the zenodo DOI. So what is the right way? We could also name @peisenha and me as the authors or just you? I need some clear advisement on that. We could also talk briefly on the phone at some point. Also I suggest we do a new release, with the whole new setup!

File: robupy/__init__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import warnings
 import socket
+import warnings
 
 # This turns of the RuntimeWarnings during production so that they do not clutter the log files.
```
Comment (peisenha) [inline robupy/__init__.py:4]: This is too machine-specific. Just remove at this point.

**Review by peisenha — APPROVED** [review]
Just one small comment below.

File: robupy/__init__.py (hunk 1 above)
Comment (MaxBlesch) [reply robupy/__init__.py:4]: Okay. Will do that in the next PR when I include Codecov badge!

**Pull request merged**